### ISSUE TITLE
[Adaptive Cards - Inputs.Json]: Screen Reader does not announce the Name of edit fields in 'Input Text Elements' card under 'Inputs.Json'.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6980]
https://microsoft.visualstudio.com/OS/_workitems/edit/35357399
